addons/monit/monit_checks_configurations/00_packetfence.tt

```diff
@@ -114,11 +114,16 @@ check process packetfence-redis_queue with pidfile /usr/local/pf/var/run/redis_q
     if failed host 127.0.0.1 port 6380 protocol redis for 5 cycles then alert
     if 3 restarts within 10 cycles then alert
 
+check process packetfence-winbind with pidfile /usr/local/pf/var/run/winbindd-wrapper.pid
+    group PacketFence
+    start program = "/usr/local/pf/bin/pfcmd service windbindd start" with timeout 60 seconds
+    stop program = "/usr/local/pf/bin/pfcmd service windbindd stop"
+    if 3 restarts within 10 cycles then alert
+
 
 [% FOREACH domain IN DOMAINS.keys %]
 check process packetfence-winbind-[% domain %] with pidfile [% DOMAINS.$domain %]
     group PacketFence
-    start program = "/usr/sbin/winbindd -D -s /etc/samba/[% domain %].conf -l /var/log/samba[% domain %]" with timeout 60 seconds
-    stop program  = "/bin/kill [% DOMAINS.$domain %]"
-    if 3 restarts within 10 cycles then alert
+    if changed ppid then alert
```

Comment:
From what I understand that would check to see if the parent process died and the child process get reparented.
However it may not be the right thing to do.
Any suggestions on the proper check?

Or do we just check the parent process?